aws/aws-service-bundler/src/main/java/software/amazon/smithy/java/aws/servicebundle/bundler/ApiStandardTerminology.java

```diff
@@ -40,8 +40,8 @@ final class ApiStandardTerminology {
             "Untag",
             "Update");
 
-    static final Set<String> READ_ONLY_API_PREFIXES = withCommonsPrefixes(READ_ONLY_TERMS);
-    static final Set<String> WRITE_API_PREFIXES = withCommonsPrefixes(WRITE_TERMS);
+    public static final Set<String> READ_ONLY_API_PREFIXES = withCommonsPrefixes(READ_ONLY_TERMS);
+    public static final Set<String> WRITE_API_PREFIXES = withCommonsPrefixes(WRITE_TERMS);
```

Comment:
Expose these through static getters if they need to be exposed. Direct field access limits our ability to change how these lists are constructed in the future.